static/scripts/audit-report/audit.ts

```diff
@@ -9,12 +9,27 @@ import { ObserverKeys, ElemInterface, QuickImport, StandardInterface, TxData, Go
 
 const interceptorID = rax.attach(axios);
 const rateOctokit = Octokit.plugin(throttling);
-let ETHERSCAN_API_KEY = "";
+
+enum ChainScan {
+  Ethereum = "etherscan.io",
+  Gnosis = "gnosisscan.io"
+}
+
+enum Chain {
+  Ethereum = "Ethereum",
+  Gnosis = "Gnosis"
+}
+
+let CHAIN: string = Chain.Ethereum
+let CHAINSCAN_API_KEY = "";
 let RPC_URL = "";
 let BOT_WALLET_ADDRESS = "";
 let GITHUB_PERSONAL_ACCESS_TOKEN = "";
 let OWNER_NAME = "";
 let REPOSITORY_NAME = "";
+interface RateLimitOptions {
+  method: string, url: string
+}
 
 const botNodeId = "BOT_kgDOBr8EgA";
 const claimUrlRegExp = /https:\/\/pay\.ubq\.fi\?claim=[a-zA-Z0-9=]+/;
```

Comment:
Technically URL parameters can be in any order. I suppose we just need to be very careful about always using the `?claim=` parameter first for any future regular expressions? Is there a more robust way to handle this?

Reply:
> Technically URL parameters can be in any order. I suppose we just need to be very careful about always using the `?claim=` parameter first for any future regular expressions? Is there a more robust way to handle this?

That wasn't added in this commit but yeah.. we handle query in URL using the `new URLSearchParams`, that way we can get the queries and see if claim and network is available no matter the order

Reply:
@pavlovcik i fixed this in the new commit.. just a quick fix